nl/guides/course-management/README.md

```diff
@@ -0,0 +1,156 @@
+
+  ![image](./staff.hidden_course_registration_link.png)
+
+  Druk op de vernieuwknop naast de [registratielink](#registratielink) om de cursus opnieuw te verbergen nadat de registratielink gedeeld werd. Daardoor wordt een nieuw token gegenereerd en wordt het oude token onbruikbaar gemaakt. De registratielink wordt meteen ook aangepast aan het nieuwe token.
+
+  ![image](./staff.hidden_course_registration_link_renew.png)
+
+Druk op de afwerkknop in de rechteronderhoek van de pagina `CURSUS AANMAKEN` om het aanmaken van een cursus met de opgegeven [eigenschappen](#cursus-eigenschappen) effectief door te voeren.
+
+![image](./staff.course_new_submit.png)
+
+Na het aanmaken van de nieuwe cursus navigeer je naar de cursuspagina, waar je kunt vaststellen dat je automatisch geregistreerd bent als cursusgebruiker en dat je aangeduid bent als [cursusbeheerder](#cursusbeheerder).
+
+![cursus aangemaakt](./staff.course_created.png)
+
+## Cursus navigeren
+
+Je kan de cursuspagina van een [cursus](#cursus) bereiken op verschillende manieren. Vanaf de landingspagina kan je cursussen zoeken door te klikken op `Meer cursussen`. Vanop eender welke pagina kan je via het hamburgermenu in de linkerbovenhoek snelkoppelingen vinden naar jouw cursusssen onder het tabblad `Cursussen` en de cursus-zoekenpagina via het gelijknamige knopje on het tabblad `Admin`. Op de zoeken-pagina kan je filteren op naam van de cursus en op het instituut waartoe de cursus behoort. Ten slotte dien je enkel nog de gewenste cursus aan te klikken om op de cursuspagina terecht te komen.
+
+![gefilterde cursussen](./staff.courses_filtered.png)
+
+## Cursus bewerken
+
+Als cursusbeheerder kan je de [eigenschappen](#cursuseigenschappen-instellen) van een cursus aanpassen door op het bewerken-icoontje te drukken bovenaan de cursuspagina.
+
+![image](./staff.course_edit_button.png)
+
+Druk op de cursus in de [navigatiebalk](../for-students#navigatiebalk) om het aanpassen te annuleren.
+
+![image](./staff.course_edit_cancel.png)
+
+Druk na het aanpassen op de afwerkknop in de rechterbovenhoek van het paneel om de nieuwe cursuseigenschappen effectief in te stellen.  Alternatief kan je ook op de `AANPASSEN` drukken onderaan het paneel.
+
+Na het afwerken navigeer je terug naar de cursuspagina waar de nieuwe cursuseigenschappen onmiddellijk van toepassing zijn.
+
+![image](./staff.course_after_edit.png)
+
+## Leerpad
+
+Als cursusbeheerder kan je voor de cursus een leerpad uitstippelen. Het leerpad wordt op de cursuspagina weergegeven onder de hoofding `Oefeningenreeksen`.Aan het leerpad kan je oefeningenreeksen toevoegen waaraan je oefeningen kunt koppelen. Voor meer uitleg over wat je allemaal kan doen, zie [oefeningenreeksen beheren](../exercise-series-management).
+
+## Cursusgebruikers beheren
+
+Deze informatie is verhuisd naar een aparte [pagina](../user-management).
```

Comment:
Be a little more helpful. e.g. Using the user icon in the navigation bar, you can get an overview of all users in the course. More information about user management can be found at x